Build/Cake/version.cake

```diff
@@ -56,6 +57,21 @@ Task("UpdateDnnManifests")
     XdtTransformConfig(file, transformFile, file);
 });
 
+Task("SetPackageVersions")
+  .IsDependentOn("SetVersion")
+  .Does(() => {
+    var packages = GetFiles("./Dnn.AdminExperience/ClientSide/*.Web/package.json");
```

Comment:
Any reason we dont just call 'lerna version @version' here? and let lerna update all the items defined in the workspaces?

Reply:
Yes Leena version fires up yarn version that fires a yarn install for each which takes forever

Reply:
But that is also what would propagate proper changes to the lock file

Reply:
Yes, this way too, yarn install still runs, but this way it runs once instead of 20 times :)